modules/ROOT/pages/guided-setup.adoc

```diff
@@ -58,10 +58,22 @@ If your Anypoint account has the organization administrator role, provide your A
 Otherwise, provide these app details:
 +
 .. Set *Client ID* to the client ID from the connected app you created.
-.. In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *IDP Issuer* string.
-.. Set *IDP Issuer* to the *IDP Issuer* string.
-.. In your Anypoint Platform organization, click *Access Management* > *Identity Providers* > *SAML 2.0* > *Edit* and copy the *Audience* string.
-.. Set *Audience* to the *Audience* string.
+.. To find the identity provider ID, run this request:
```

Comment:
Looking at the context for the PR, I see that this is a really long task, with only two first-level steps. This indicates some sort of organizational issue that should be addressed.

Reply:
This task has the three first-level steps "Select your organization's Anypoint management plane...", "Create a connected app...", and "Authenticate and authorize the connected app...". However, three may still be too low. I will review the organization here in a separate Jira story.